DbUp.Support.SqlServer.Scripting.Console/Program.cs

```diff
@@ -19,7 +21,17 @@ static int Main(string[] args)
                 .LogToConsole()
                 .Build();
 
-        ScriptingUpgrader upgradeScriptingEngine = new ScriptingUpgrader(connectionString, engine);
+        Options options = new Options()
+        {
+            ScriptingOptions = new ScriptingOptions()
```

Comment:
Rather than this being the default, and unable to be changed, can you use the same pattern as `scriptAllDefinitions` where it can be passed in as a arg?

See:
https://github.com/bradymholt/dbup-sqlserver-scripting/blob/master/DbUp.Support.SqlServer.Scripting/ScriptingUpgrader.cs#L105